packages/realm/src/schema/types.ts

```diff
@@ -27,7 +27,7 @@ export type DefaultObject = Record<string, unknown>;
 export type Constructor<T = unknown> = { new (...args: any): T };
 export type RealmObjectConstructor<T extends RealmObject = RealmObject> = {
   new (...args: any): T;
-  schema: ObjectSchema;
+  schema?: ObjectSchema;
```

Comment:
Adding a comment to why this is needed would also be useful for our future selves 🙂

Reply:
Good point!  Done 👍🏼